src/main/java/mate/academy/hw01/Matrix.java

```diff
@@ -0,0 +1,42 @@
+package mate.academy.hw01;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class Matrix {
+    static Integer[][] arr;
+
+    public static void main(String[] args) {
+        arr = createArray(3, 3);
+        System.out.println(Arrays.deepToString(arr));
+        foundMinMax(arr);
+    }
+
+    private static Integer[][] createArray(Integer m, Integer n) {
```

Comment:
Why do you use wrappers everywhere ? Not an issue , just explain why

Reply:
You dont use any features of wrappers why not primitives ?

Reply:
Well, when I use the functions of the packers, I will use the wrappers, in other cases I will use primitives. Use the wrappers was just a desire, an experiment

Reply:
It uses more memory, it can be null  and you do not check this. This performs unboxing/autoboxing when some calculations are performed.  You should consider this when choosing to use wrapers.